homeassistant/components/google_assistant/__init__.py

```diff
@@ -76,6 +77,7 @@ def _check_report_state(data):
             vol.Optional(CONF_SECURE_DEVICES_PIN): str,
             vol.Optional(CONF_REPORT_STATE, default=False): cv.boolean,
             vol.Optional(CONF_SERVICE_ACCOUNT): GOOGLE_SERVICE_ACCOUNT,
+            vol.Optional(CONF_LOCAL_FULFILLMENT, default=False): cv.boolean,
```

Comment:
Any specific reason why this should be off by default?

Reply:
The reason I put it like that is twofold:
- you need to upload .js files in your google actions configuration before this will work
- the currently present code made it look like this is something that should not always be on (why else is it not already turned on), so I went with that approach

I am fine with enabling it by default or removing the config entirely, though I guess I would need to test whether that breaks anything for the cloud SYNC. I am not expecting difficulties though.

Reply:
> (why else is it not already turned on)

Maybe, because the library feature was built later and wasn't built in this part of the code yet?

I think if it can be detected or just enabled always without repercussions is always better than introducing more configuration complexity.

Reply:
Now it is not configurable and enabled by default. According to the [Google documentation](https://developers.google.com/assistant/smarthome/develop/process-intents#sync-response) the response should always be able to contain otherDeviceIds and customData, which are the two things that are added to the response if local fulfillment is enabled.

Reply:
I tested it both with and without the local fulfillment application loaded to the Google home device, and as far as I can tell everything works as expected in both scenarios.